### ISSUE TITLE
yExporter

### ISSUE THREAD

**Issue by 0xBasically** [yearn/budget#126]
### Scope

This is a budget request for maintaining all exporter-related tasks. The exporter is a service that powers [yearn.vision](https://yearn.vision), calculates APYs for all our networks, serves data for various internal and external front-ends, and is used to visualize partner, treasury, and transaction data.

### Plan

As of today, BobTheBuilder and Crebsy are taking care of most of the code maintenance, bug fixes, and are doing code reviews on a regular basis. 0xBasic is reviewing PRs and is maintaining grafana dashboards for the exporter. The plan is to keep this team as-is and by fixing the tests and cleaning the codebase, to make it more easy for new contributors to start to code with it.

### High priority tasks
* fix tests
* finish dank async refactoring
* fix treasury exporter for fantom
* fix curve forward-looking apy for all chains: add velo and bal apys

### Medium priority tasks:
* consolidate data layer
* refactor monolithic code into smaller services/libs
* replace eth-brownie with ape once ape has async support

### Public group
#yearn-exporter in [discord](discord.yearn.finance)

### Deadline

2023-11-01

### People

- BobTheBuilder
- Cresby
- 0xBasic

### Money

This is our request per month:

| Type    | Dev           | DAI      | YFI     | 
| ------- | ------------- | -------- | ------- |
| Infrastructure | ---------------------- | 525     | 0       | 
| DEV     | crebsy        | 7500     | 0.75    |  
| DEV     | BobTheBuilder | 7500     | 0.75    |   
| DEV     | 0xBasic       | 7500     | 0.75    |    
| ============ | ============= | ======== | ======= |     
| ---------------------- | TOTAL         | 23025    | 2.25    |  

### Amount

23025 DAI & 2.25 YFI

### Wallet address

TBD

### Reporting

Monthly

**Comment by 0xBasically** [comment]
update its a safe

**Comment by 0xBasically** [comment]
For the month of June, the team has been working on 

## Apys

Fix errors causing s3 to crash

https://github.com/yearn/yearn-exporter/pull/525 
https://github.com/yearn/yearn-exporter/pull/527 

### Add Velo

https://github.com/yearn/yearn-exporter/pull/531 
https://github.com/yearn/yearn-exporter/pull/530 

## Master fixes/updates

https://github.com/yearn/yearn-exporter/pull/522 
https://github.com/yearn/yearn-exporter/pull/520
https://github.com/yearn/yearn-exporter/pull/518 
https://github.com/yearn/yearn-exporter/pull/515 

## The Big One Temp: Async Refactoring

testing fixes, updates, refactorings almost ready to merge

https://github.com/yearn/yearn-exporter/pull/510 

## High Priority Tasks 
- [x] fix treasury exporter for fantom (fixed in temp) 
- [ ] fix tests (on hold till after temp is merged) 
- [x] finish dank async refactoring (fixed in temp)
- fix curve forward-looking apy for all chains: add velo and bal apys
  - [x] Velo
  - [ ] Balancer (about to start)

## Medium priority tasks:
- [ ] consolidate data layer (on hold till temp is merged)
- [ ] refactor monolithic code into smaller services/libs (on hold till temp is merged)
- [ ] replace eth-brownie with ape once ape has async support (waiting on ape)

**Comment by 0xBasically** [comment]
Based on feedback from yBudget we are limiting this BR to 3m thus it will end on 7/31/2023
